MantidQt/CustomInterfaces/src/Reflectometry/QtReflSettingsTabView.cpp

```diff
@@ -0,0 +1,53 @@
+#include "MantidQtCustomInterfaces/Reflectometry/QtReflSettingsTabView.h"
+#include "MantidQtCustomInterfaces/Reflectometry/ReflSettingsTabPresenter.h"
+
+namespace MantidQt {
+namespace CustomInterfaces {
+
+//----------------------------------------------------------------------------------------------
+/** Constructor
+* @param parent :: [input] The parent of this widget
+*/
+QtReflSettingsTabView::QtReflSettingsTabView(QWidget *parent) {
+
+  initLayout();
+
+  m_presenter.reset(new ReflSettingsTabPresenter(this));
```

Comment:
I would move this into the initializer list. That way you shouldn't need the reset.